app/build.gradle.kts

```
@@ -0,0 +1,147 @@
+            proguardFiles(
+                getDefaultProguardFile("proguard-android.txt"),
+                "proguard-rules.pro"
+            )
+            resValue("string", "app_name", "Catima")
+        }
+        debug {
+            applicationIdSuffix = ".debug"
+            resValue("string", "app_name", "Catima Debug")
+        }
+    }
+
+    buildFeatures {
+        buildConfig = true
+        viewBinding = true
+    }
+
+    bundle {
+        language {
+            enableSplit = false
+        }
+    }
+
+    compileOptions {
+        encoding = "UTF-8"
+
+        // Flag to enable support for the new language APIs
+        isCoreLibraryDesugaringEnabled = true
+
+        sourceCompatibility = JavaVersion.VERSION_11
+        targetCompatibility = JavaVersion.VERSION_11
+    }
+
+    sourceSets {
+        getByName("test") {
+            resources.srcDirs("src/test/res")
+        }
+    }
+
+    // Starting with Android Studio 3 Robolectric is unable to find resources.
+    // The following allows it to find the resources.
+    testOptions.unitTests.isIncludeAndroidResources = true
+    tasks.withType<Test>().configureEach {
+        testLogging {
+            events("started", "passed", "skipped", "failed")
+        }
+    }
+
+    lint {
+        lintConfig = file("lint.xml")
+    }
+}
+
+dependencies {
+
+    // AndroidX
+    implementation("androidx.appcompat:appcompat:1.6.1")
+    implementation("androidx.constraintlayout:constraintlayout:2.1.4")
+    implementation("androidx.exifinterface:exifinterface:1.3.6")
+    implementation("androidx.palette:palette:1.0.0")
+    implementation("androidx.preference:preference:1.2.1")
+    implementation("com.google.android.material:material:1.9.0")
+    implementation("com.github.yalantis:ucrop:2.2.8")
+    coreLibraryDesugaring("com.android.tools:desugar_jdk_libs:2.0.3")
+
+    // Splash Screen
+    implementation("androidx.core:core-splashscreen:1.0.1")
+
+    // Third-party
+    implementation("com.journeyapps:zxing-android-embedded:4.3.0@aar")
+    implementation("com.google.zxing:core:3.5.2")
+    implementation("org.apache.commons:commons-csv:1.9.0")
+    implementation("com.jaredrummler:colorpicker:1.1.0")
+    implementation("net.lingala.zip4j:zip4j:2.11.5")
+
+    // SpotBugs
+    implementation("io.wcm.tooling.spotbugs:io.wcm.tooling.spotbugs.annotations:1.0.0")
+
+    // Testing
+    testImplementation("androidx.test:core:1.5.0")
+    testImplementation("junit:junit:4.13.2")
+    testImplementation("org.robolectric:robolectric:4.10.3")
+}
+
+tasks.withType<SpotBugsTask>().configureEach {
+    description = "Run spotbugs"
+    group = "verification"
+
+    //classes = fileTree("build/intermediates/javac/debug/compileDebugJavaWithJavac/classes")
+    //source = fileTree("src/main/java")
+    //classpath = files()
+
+    reports.maybeCreate("xml").required.set(false)
+    reports.maybeCreate("html").required.set(true)
+}
+
+tasks.register("copyRawResFiles", Copy::class.java) {
```

Any reason to use `.java` here?

```suggestion
tasks.register("copyRawResFiles", Copy::class) {
```

---

Out of habit, dropped.